packages/lab/src/table/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./Table";
+export * from "./TBody";
+export * from "./TD";
+export * from "./TFoot";
+export * from "./TH";
+export * from "./THead";
+export * from "./TR";
```

Comment:
@honey-chang this means we can remove 7 components from mosaic, which has the exact same components, except styling which will change. They should also be able to plug straight into the MDX Provider of both internal and external site.

Once released, we can update Mosaic and then pull these back into the sites.

We should do this before we push them to core, to how well they work with JamStack.

@honey-chang from your pov, do these styles align to your expectations or do we need to consider anything?